Sources/SwiftSyntax/Syntax.swift

```diff
@@ -766,3 +663,6 @@ extension RawSyntaxNodeProtocol {
     return Syntax(raw: raw).recursiveDescription
   }
 }
+
+@available(*, unavailable, message: "use 'Syntax' instead")
+public struct SyntaxNode {}
```

Comment:
Is this useful in any way or just an artifact from your local development?

Reply:
I left this only because `SyntaxNode` was a public type. Since the initializer was not public, I don't anticipate anyone is using this. But still.